en/docs/api-management/api-policies.md

```diff
@@ -58,13 +58,13 @@ To deploy the API follow the steps below:
 Choreo allows you to implement an API policy as a Ballerina project and attach it to an API proxy component. 
 
 !!! info
-    Supported Ballerina versions: between 2201.3.1 and 2201.5.1.
+    Supported Ballerina versions: 2201.5.1 
```

Comment:
Since we're mentioning just one version now, shall we say:
Supported Ballerina version: 2201.5.1